src/org/labkey/test/util/DataRegionTable.java

```diff
@@ -1306,6 +1306,20 @@ public void setContainerFilter(ContainerFilterType filterType)
         getViewsMenu().clickSubMenu(true, "Folder Filter", filterType.getLabel());
     }
 
+    public String getContainerFilter()
+    {
+        getViewsMenu().openMenuTo("Folder Filter", "Folder Filter");
+        var visibleFilterMenuItemOptions = getViewsMenu().findVisibleMenuItems();
+        for (WebElement el : visibleFilterMenuItemOptions)
+        {
+            if (Locator.tagWithClass("i", "fa-check-square-o").existsIn(el))
+            {
+                return el.getText();   // would be nice if enum.valueOf() allowed us to get the enum constant from its text but oh java you crazy
```

Comment:
nit: While I appreciate the sentiment this comment isn't very useful.

Reply:
Fair enough.  I just put it there as a way to fish for suggestions- I mean I could implement a 'fromLabel' method on that enum that streams over its possible label values and returns the constant value for the matching one (or null), but sometimes Trey will have a trick or helper somewhere that solves the problem.  I can remove the comment if we're ok with returning a string instead of an enum

Reply:
No tricks for this one. Adding a `ContainerFilterType.fromLabel` method seems reasonable though. We have such methods in several test enums.